server/controllers/Sessions.js

```diff
@@ -0,0 +1,78 @@
+import bcrypt from 'bcryptjs';
+import {
+  findUser,
+  // generateToken,
+  serverResponse,
+  serverError
+  // expiryDate
+} from '../helpers';
+// import models from '../database/models';
+
+// const { Session } = models;
+// const { findUser } = helpers;
+// const mobileDeviceIndicator = [
+//   'mobile',
+//   'andriod',
+//   'iphone',
+//   'tablet',
+//   'ipad',
+//   'ipod'
+// ];
+
+/**
+ *
+ *
+ * @class Sessions
+ */
+class Sessions {
+  /**
+   *
+   *
+   * @static
+   * @param {object} req - request object
+   * @param {object} res - response object
+   * @memberof User
+   * @returns {json}  object
+   */
+  static async create(req, res) {
+    try {
+      const { userLogin, password } = req.body;
+      if (!/\D/.test(userLogin)) {
```

Comment:
I don't think we need this, just call `toString` on that variable if you must accept a string, regexes are computationally expensive